docs/en/latest/plugins/authz-keycloak.md

```diff
@@ -48,7 +48,7 @@ Refer to [Authorization Services Guide](https://www.keycloak.org/docs/latest/aut
 | token_endpoint                               | string        | False    |                                               | https://host.domain/auth/realms/foo/protocol/openid-connect/token  | An OAuth2-compliant token endpoint that supports the `urn:ietf:params:oauth:grant-type:uma-ticket` grant type. If provided, overrides the value from discovery.                                                                                       |
 | resource_registration_endpoint               | string        | False    |                                               | https://host.domain/auth/realms/foo/authz/protection/resource_set  | A UMA-compliant resource registration endpoint. If provided, overrides the value from discovery.                                                                                                                                                      |
 | client_id                                    | string        | True     |                                               |                                                                    | The identifier of the resource server to which the client is seeking access.                                                                                                                                                                         |
-| client_secret                                | string        | False    |                                               |                                                                    | The client secret, if required.                                                                                                                                                                                                                       |
+| client_secret                                | string        | False    |                                               |                                                                    | The client secret, if required. You can use APISIX secret to store and reference this value. APISIX currently supports storing secrets in two ways. [Environment Variables](#use-environment-variables-to-manage-secrets) and [HashiCorp Vault](#use-vault-to-manage-secrets)                                                                                                                                                                                                                       |
```

Comment:
The links are wrong. And please add an example as well.

Reply:
I fixed the doc link. Though the lint points to the secret page which has the example in detail. So I think it will be redundant to add it again here.

Reply:
Does this PR introduce support to use env vars for accessing secrets? If yes, please add tests as well.

Regarding the docs, it would be nice to have an example like this:

(existing example that uses hard-coded value for secrets)

You can also set secrets using the secret resource to avoid having sensitive information in plain text <-- add this line

(add a basic configuration here)

See (add link to secrets.md documentation) for more information.

Reply:
@shreemaan-abhishek Though secret is the APISIX abstraction. Behind it can be vault or env variable or anything. So I thought just testing with any one of them should work. Okay I will add tests for env variable as well.

Reply:
okay i will modify the test case